simple_kafka_client/SimpleKafkaClient/Consumer.php

```diff
@@ -0,0 +1,90 @@
+<?php
+declare(strict_types=1);
+
+namespace SimpleKafkaClient;
+
+use SimpleKafkaClient;
+
+class Consumer extends SimpleKafkaClient
+{
+    /**
+     * Consumer constructor.
+     * @param Configuration $configuration
+     * @throws Exception
+     */
+    public function __construct(Configuration $configuration) {}
+
+    /**
+     * @param array|null $topics
+     * @throws Exception
+     */
+    public function assign(?array $topics): void {}
+
+    /**
+     * @return array
+     * @throws Exception
+     */
+    public function getAssignment(): array {}
+
+    /**
+     * @param array $topics
```

Comment:
```suggestion
     * @param Topic[] $topics
```
Maybe this should be everywhere ?